child_compassion/model/child_compassion.py

```diff
@@ -328,6 +338,20 @@ def _get_last_pictures(self, cr, uid, child_id, context=None):
 
         return pic_id
 
+    def _create_empty_case_study(self, cr, uid, child, context=None):

```

Comment:
instead of passing the whole child record, you could simply pass child_id value.